packages/components/src/Illustration/IllustrationTypes.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+import { type StylePropType } from '@kiwicom/universal-components';
+
+export type IllustrationProps = {|
+  source: string,
```

Comment:
What about using `name` instead of `source` to be consistent with Orbit ( https://orbit.kiwi/components/illustration/ ).

Also we can use union types similar to https://github.com/kiwicom/orbit-components/blob/master/src/Illustration/index.js.flow (but just use currently implemented illustrations, because of the mobile version).

Reply:
Good idea!